Source/WebCore/style/StyleAdjuster.cpp

```diff
@@ -591,6 +599,11 @@ void Adjuster::adjust(RenderStyle& style, const RenderStyle* userAgentAppearance
         if (m_element == m_document.activeModalDialog() && !hasInertAttribute(m_element))
             style.setEffectiveInert(false);
 
+#if ENABLE(INTERACTION_REGIONS_IN_EVENT_REGION)
+        if (m_element == m_document.fullscreenManager().fullscreenElement())
```

Comment:
We do want to make fullscreen elements with the inert attribute inert as well. So ` && !hasInertAttribute(m_element)` is needed.